File: TwoSum_V1.php
<?php
  class Solution_V1
  {
      /**
       * @param array $nums
       * @param Integer $target
       * @return array
       */

      public function twoSum(array $nums, $target)
      {
          $count  =  count($nums);
          if ($count == 0) return [];

          for ($i = 0; $i < $count; $i++) {
              $value  = $nums[$i];

              if ($a = array_keys($nums, $target - $value)) {
                  return array($a[0], $i);
              }
          }
          return [];
      }
  }

  $s        = new Solution_V1();
  $sum      = [2,7,11,115];
  $target   = 17;

  print_r($s->twoSum($sum, $target));

File: Leecode-1/TwoSum_V1.go
package main

func twoSumV1(nums []int, target int) []int {

	if len(nums) == 0 {
		return []int{}
	}

	for i , v := range nums {
		for k := i + 1; k < len(nums); k ++ {
			if target - v == nums[k] {
				return []int{i, k}
			}
		}
	}

	return []int{}
}